### PR TITLE
[zh] translate better-docs-ux-with-docsy

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/36383

https://kubernetes.io/blog/2020/06/better-docs-ux-with-docsy/ does not have chinese version

so need to translate it into Chinese

### CODE REVIEW

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: Docsy 带来更好的 Docs UX
+date: 2020-06-15
+slug: better-docs-ux-with-docsy
+url: /blog/2020/06/better-docs-ux-with-docsy
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:6]: This makes the English version inaccessible.
```suggestion
url: /zh-cn/blog/2020/06/better-docs-ux-with-docsy
```

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+<!--
+*Editor's note: Zach is one of the chairs for the Kubernetes documentation special interest group (SIG Docs).*
+-->
+**编者注：Zach 是 Kubernetes 文档特别兴趣小组（SIG Docs）的主席之一。**
+
+<!--
+I'm pleased to announce that the [Kubernetes website](https://kubernetes.io) now features the [Docsy Hugo theme](https://docsy.dev).
+-->
+我很高兴地宣布 [Kubernetes 网站](https://kubernetes.io) 现在采用了 [Docsy Hugo 主题](https://docsy.dev)。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:30]:
```suggestion
我很高兴地宣布 [Kubernetes 网站](https://kubernetes.io)现在采用了 [Docsy Hugo 主题](https://docsy.dev)。
```

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+看到这个目标触手可及，真是令人兴奋。
+
+<!--
+One of SIG Docs' tech leads, [Karen Bradshaw](https://github.com/kbhawkey) did a lot of heavy lifting to fix a wide range of site compatibility issues, 
+including a fix to the last of our [legacy pieces](https://github.com/kubernetes/website/pull/21359) when we [migrated from Jekyll to Hugo](2018-05-05-hugo-migration/) in 2018.
+Our other tech leads, [Tim Bannister](https://github.com/sftim) and [Taylor Dolezal](https://github.com/onlydole) provided extensive reviews.
+-->
+作为 SIG Docs 的技术负责人之一，[Karen Bradshaw](https://github.com/kbhawkey) 做了很多繁重的工作来解决广泛的网站兼容性问题，
+包括在 2018 年我们从 [Jekyll 迁移到 Hugo](2018-05-05-hugo-migration/) 时修复的最后一个[遗留问题](https://github.com/kubernetes/website/pull/21359)。
+我们的其他技术负责人, [Tim Bannister](https://github.com/sftim) 和 [Taylor Dolezal](https://github.com/onlydole) 提出过很多意见。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:72]:
```suggestion
我们的其他技术负责人，[Tim Bannister](https://github.com/sftim) 和 [Taylor Dolezal](https://github.com/onlydole) 提出过很多意见。
```

**Review by Sea-n — COMMENTED** [review]
With `url` metadata set to `/blog/...`, this PR replaced English page this Mandarin version:

https://deploy-preview-36384--kubernetes-io-main-staging.netlify.app/blog/2020/06/better-docs-ux-with-docsy/

**Comment by twilight0620** [comment]
I have modified them, thanks for reviewing

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+<!--
+The Docsy theme improves the site's organization and navigability, and opens a path to improved API references. 
+After over 4 years with few meaningful UX improvements, Docsy implements some best practices for technical content. 
+The theme makes the Kubernetes site easier to read and makes individual pages easier to navigate. 
+It gives the site a much-needed facelift.
+-->
+Docsy 主题改进了网站的组织和可导航性，并开辟了改进 API 参考的途径。
+在 4 年多的时间里，很少有有意义的用户体验改进，Docsy 实施了一些技术内容的最佳实践。
+该主题使 Kubernetes 网站更易于阅读，并使各个页面更易于导航。
+它对该网站进行了必需的翻新。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:41]:
```suggestion
它大大改进了网站所需的外在形象。
```

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+-->
+我很高兴地宣布 [Kubernetes 网站](https://kubernetes.io)现在采用了 [Docsy Hugo 主题](https://docsy.dev)。
+
+<!--
+The Docsy theme improves the site's organization and navigability, and opens a path to improved API references. 
+After over 4 years with few meaningful UX improvements, Docsy implements some best practices for technical content. 
+The theme makes the Kubernetes site easier to read and makes individual pages easier to navigate. 
+It gives the site a much-needed facelift.
+-->
+Docsy 主题改进了网站的组织和可导航性，并开辟了改进 API 参考的途径。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:38]:
```suggestion
Docsy 主题改进了网站的组织结构和导航性能，并开辟了改进 API 参考的途径。
```

File: content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+在 4 年多的时间里，很少有有意义的用户体验改进，Docsy 实施了一些技术内容的最佳实践。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2020-06-15-better-docs-ux-with-docsy.md:39]:
```suggestion
在 4 年多的时间里，尽管对用户体验方面的改进不多，但 Docsy 针对技术内容实现一些最佳实践。
```

**Pull request merged**